pkg/executables/helm.go

```diff
@@ -61,13 +62,27 @@ func NewHelm(executable Executable, opts ...HelmOpt) *Helm {
 	return h
 }
 
-func (h *Helm) Template(ctx context.Context, ociURI, version, namespace string, values interface{}, kubeVersion string) ([]byte, error) {
+func (h *Helm) Template(
```

Comment:
I mentioned it in the PR first comment:
`(Some reformatting done by golines which I can remove if need be but I think we should use golines)`

Regardless of preference, using an automated approach is useful to avoid changes hidden by reformatting. This is included because I modified this file and `golines` made the modifications and people using different formats will be a source of similar issues going forward.

Reply:
As I see it there are two issues here. The first is the use of the golines tool. I think this is the wrong way to introduce the tool. I think it should be brought up with and discussed with the team. If the team decides to use it, someone should then reformat all the files in one fell swoop, so that reviewers know that the only changes expected are non-functional whitespace changes.

The second issue is that this change is unrelated to the subject of the PR. These kinds of changes shouldn't be included in PRs that have functional changes, which, as you wrote (and yes, I read what you wrote), you know, yet you did it anyway (why?). I'm not asking you to remove this change; I've already looked at it, verified that it's a whitespace only change and unrelated. My comment was meant to reinforce what I'm guessing you already know; that these aren't good changes to include, and I'd be grateful if we all tried not to include them in the future.

As an aside, you wrote:

> using an automated approach is useful to avoid changes hidden by reformatting

Your automated tool is the culprit here, introducing "false" changes that distract from what really is changing. Yes, people having different settings will cause these problems. Right now, the person with different settings... is you. 😛 

Nothing automated is required. People just need to not alter the formatting of things they aren't changing, and there won't be any such problems.